packages/prerender/src/runPrerender.tsx

```diff
@@ -67,7 +67,7 @@ export const runPrerender = async ({
   // This is set by webpack by the html plugin
   const renderOutput = indexContent.replace(
     '<server-markup></server-markup>',
```

Comment:
I actually thought we would remove `server-markup` and put `<div id="redwood-app-prerendered" />` in `index.html` and then insert/ replace that with `<div id="redwood-app-prerendered">${componentAsHtml}</div>`